src/vs/workbench/api/common/menusExtensionPoint.ts

```diff
@@ -367,7 +373,8 @@ commandsExtensionPoint.setHandler(extensions => {
 			title,
 			category,
 			precondition: ContextKeyExpr.deserialize(enablement),
-			iconLocation: absoluteIcon
+			iconLocation: absoluteIcon,
+			iconClassName: iconClassName
```

Comment:
`iconClassName` that shouldn't be here... I know it's not you but this stands out. We had it and spend quite some effort to get rid of this because its incompatible to Mac touch bars @misolori You seem to have added this and this is a FYI. 

For this PR it means that we need to drag the URI along until it's being rendered

Reply:
iconClassName removal -> https://github.com/microsoft/vscode/issues/85673